prow/scripts/lib/log.sh

```diff
@@ -11,34 +11,46 @@ function log::date {
     date +"%Y/%m/%d %T %Z"
 }
 
+# log::banner prints message with INFO level in banner form for easier spotting in log files
+#
+# Arguments:
+#   $1 - Message
+function log::banner {
+  local logdate
+  logdate=$(log::date)
+  echo -e "${INVERTED}${logdate} [INFO] *************************************************************************************${NC}"
+  echo -e "${INVERTED}${logdate} [INFO] * $* ${NC}"
+  echo -e "${INVERTED}${logdate} [INFO] *************************************************************************************${NC}"
+}
+
 # log::info prints message with info level
 #
 # Arguments:
 #   $1 - Message
 function log::info {
-    echo -e "${INVERTED}$(log::date) [INFO] ${1}${NC}"
+    echo -e "${INVERTED}$(log::date) [INFO] $* ${NC}"
```

Comment:
Maybe we should use something like `printf`?
Also the comment above this methdo is out of date, because all arguments are messages.

Reply:
fixed the comments, but i did not change to printf as it did not improve readability of this code imho